### PR TITLE
Fixed (I think) the LDC tool so it works on the production server.

### PR DESCRIPTION
**Pull request by hover2pi**
This should fix the LDC tool. I think it was just a type error (int should have been float) in the form validation for the LDC.

### CODE REVIEW

**Comment by hover2pi** [comment]
I also updated the form validation messages in the LDC to only show the first error, not every single error when the form is submitted.

File: exoctk/exoctk_app/templates/limb_darkening.html [hunk 1]
```diff
@@ -93,10 +93,10 @@ <h1>Limb Darkening Calculator</h1>
                     <div class="col-sm-10">
                         {% for subfield in form.modeldir %}
                             <div class='radio'>
-                                {% if subfield.label.text == 'Phoenix ACES' %}
-                                    {{ subfield(style='margin-left:0px;', id='aces', checked='checked') }}{{ subfield.label }}
+                                {% if subfield.label.text == 'Kurucz ATLAS9' %}
+                                    {{ subfield(style='margin-left:0px;', id='atlas9', checked='checked') }}{{ subfield.label }}
                                 {% else %}
-                                    {{ subfield(style='margin-left:0px;', id='atlas9') }}{{ subfield.label }}
+                                    {{ subfield(style='margin-left:0px;', id='aces') }}{{ subfield.label }}
```
Comment (julesfowler) [inline exoctk/exoctk_app/templates/limb_darkening.html]: @hover2pi Why switch from ACES --> ATLAS?

File: exoctk/exoctk_app/form_validation.py [hunk 2]
```diff
@@ -52,12 +52,12 @@ class LimbDarkeningForm(BaseForm):
...
-    feh = DecimalField('feh', default=0.0, validators=[InputRequired('A surface gravity is required!'), NumberRange(min=feh_rng[0], max=feh_rng[1], message='Metallicity must be between {} and {}'.format(*feh_rng))])
+    teff = DecimalField('teff', default=3500, validators=[InputRequired('An effective temperature is required!'), NumberRange(min=float(teff_rng[0]), max=float(teff_rng[1]), message='Effective temperature must be between {} and {} for this model grid'.format(*teff_rng))])
+    logg = DecimalField('logg', default=4.5, validators=[InputRequired('A surface gravity is required!'), NumberRange(min=float(logg_rng[0]), max=float(logg_rng[1]), message='Surface gravity must be between {} and {} for this model grid'.format(*logg_rng))])
+    feh = DecimalField('feh', default=0.0, validators=[InputRequired('A surface gravity is required!'), NumberRange(min=float(feh_rng[0]), max=float(feh_rng[1]), message='Metallicity must be between {} and {} for this model grid'.format(*feh_rng))])
```
Comment (julesfowler) [inline exoctk/exoctk_app/form_validation.py]: Float 4 days.

File: exoctk/exoctk_app/app_exoctk.py [hunk 3]
```diff
@@ -158,9 +158,8 @@ def limb_darkening():
         form.modeldir.data = [j for i, j in form.modeldir.choices if i == form.modeldir.data][0]
 
         # Grism details
-        if '.G' in form.bandpass.data.upper() and 'GAIA' not in form.bandpass.data.upper():
-            kwargs = {'n_bins': form.n_bins.data, 'pixels_per_bin': form.n_pix.data,
-                      'wl_min': form.wave_min.data*u.um, 'wl_max': form.wave_max.data*u.um}
+        if ('.G' in form.bandpass.data.upper() and 'GAIA' not in form.bandpass.data.upper()) or form.bandpass.data.lower() == 'tophat':
+            kwargs = {'n_bins': form.n_bins.data, 'wave_min': form.wave_min.data*u.um, 'wave_max': form.wave_max.data*u.um}
```
Comment (julesfowler) [inline exoctk/exoctk_app/app_exoctk.py]: @hover2pi It looks like you renamed these kwargs from `wl_min` --> `wave_min`, etc. 
I don't see a change accompanying that -- was this part of the existing bug?

Comment (hover2pi) [reply exoctk/exoctk_app/app_exoctk.py]: It was an additional bug that was making the tophat and grism filters fail. It's the argument name that gets passed to the svo_filters.Filter class.

File: exoctk/exoctk_app/form_validation.py (hunk 2 above)
Comment (hover2pi) [reply exoctk/exoctk_app/form_validation.py]: We're float rich! This is what was causing the server error.

File: exoctk/exoctk_app/templates/limb_darkening.html (hunk 1 above)
Comment (hover2pi) [reply exoctk/exoctk_app/templates/limb_darkening.html]: ATLAS9 is the default model grid but was second in the list of radio buttons, which seemed silly. So I switched them.

**Comment by hover2pi** [comment]
@julesfowler If you approve, I can merge and deploy to tlexo!

**Pull request merged**